pkgs/by-name/pd/pdftricks/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  pkg-config,
+  vala,
+  gtk3,
+  wrapGAppsHook3,
+  cmake,
+  pantheon,
+  ninja,
+  ghostscript,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "pdftricks";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    repo = "pdftricks";
+    owner = "muriloventuroso";
+    tag = finalAttrs.version;
+    hash = "sha256-l4Xg4Uk520qoaEo8UxdLE8MfpVkRj/bpGBzL5HwdDUo=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    pkg-config
+    vala
+    wrapGAppsHook3
+    ninja
+    cmake
+  ];
+
+  postPatch = ''
+    substituteInPlace data/meson.build \
+      --replace-fail "'desktop'," "" \
+      --replace-fail "'appdata'," ""
+  '';
+
+  buildInputs = [
+    gtk3
+    pantheon.granite
+    ghostscript
+  ];
+
+  dontUseCmakeConfigure = true;
+
+  meta = {
+    description = "Simple, efficient application for small manipulations in PDF files using Ghostscript";
+    homepage = "https://github.com/muriloventuroso/pdftricks";
+    changelog = "https://github.com/muriloventuroso/pdftricks/releases";
+    license = lib.licenses.gpl3Plus;
+    maintainers = with lib.maintainers; [ theobori ];
+    platforms = lib.platforms.unix;
```

Comment:
I think this package only works on linux.

(`pantheon.granite` only supports `lib.platforms.linux`)